test-tap/reporters/default.regular.v16.log

```diff
@@ -108,6 +111,12 @@
   Error: Can’t catch me
 
   [90m› passes (test-tap/fixture/report/regular/unhandled-rejection.cjs:4:17)[39m
+  [90m› Test.callFn (file://lib/test.js:507:21)[39m
+  [90m› Test.run (file://lib/test.js:520:23)[39m
+  [90m› Runner.runSingle (file://lib/runner.js:285:33)[39m
+  [90m› Runner.runTest (file://lib/runner.js:361:30)[39m
+  [90m› async file://lib/runner.js:526:21[39m
+  [90m› async Runner.start (file://lib/runner.js:534:15)[39m
 
 ---tty-stream-chunk-separator
   [1mUnhandled rejection in unhandled-rejection.cjs[22m
```

Comment:
I tried updating the .log files by running `UPDATE_REPORTER_LOG=true npx tap test-tap/reporters/*` inside a docker container. I chose a docker container so I could change the node version. For node v16, I used the node-16 image.

The .log file generated inside the docker container is not the same as the logs generated during the workflow run. All the lines in green are not generated when the workflow runs. Thus failing the pipeline.

#### How can I update the .log files for different node versions?

I hope the above makes sense. 😅